depends/patches/qt/utc_from_string_no_optimize.patch

```diff
@@ -0,0 +1,84 @@
+Modify optimisation flags for various functions.
```

Comment:
Nit: I guess this should mention `std::sort` too? Or maybe even put that into a separate patch?

Reply:
Maybe we can shuffle this all around again, the next time we bump Qt (new LTS has just been released), and have to inevitably rebase/update all these patches?